CHANGELOG.md

```diff
@@ -17,6 +17,10 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+## [7.63.8] - 2024-10-21
+### Added
+- The filters `And`, `Or` and `Not` can now be applied by using the operators `&`, `|` and `~` .
```

Comment:
```suggestion
- Filters can now be combined using `And` and `Or` by using the operators `&` and `|`.
- Filters can now be negated by using the `~` operator (instead of using the `Not` filter)
```